README.md

```diff
@@ -1,11 +1,11 @@
 # rpi-eeprom
-This repository contains the scripts and pre-compiled binaries used to create the `rpi-eeprom` package which is used to update the Raspberry Pi 4 bootloader and VLI USB xHCI controller EEPROMs.
+This repository contains the scripts and pre-compiled binaries used to create the `rpi-eeprom` package which is used to update the Raspberry Pi 4B, 400 and CM4 bootloader and VLI USB xHCI controller EEPROMs.
 
 # Support
 Please check the Raspberry Pi [general discussion forum](https://www.raspberrypi.org/forums/viewforum.php?f=63) if you have a support question. 
 
 # Reset to factory defaults
-The [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/) provides GUI for creating a GUI to update the bootloader and configure the default boot-mode.
+We recommend using [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/) to reset your EEPROMs: this will install the latest bootloader and USB controller EEPROM images, and reset the configuration back to the default.
```

Comment:
LOL, I guess the original was _supposed_ to say "...provides a GUI for creating an SD-card to update...".

Reply:
BTW, the old version mentioned "configure the default boot-mode" but the new version doesn't mention that?
Perhaps it's just me, but "reset your EEPROMs" sounds like an odd phrase, and specifically mentioning the "USB controller EEPROM" feels like an unnecessary level of detail?
I'm struggling to come up with anything "better" than the phrasing used by the original though :confused: 

"We recommend using Raspberry Pi Imager to update the bootloader (which restores the default configuration) and change the default boot mode."

Reply:
> BTW, the old version mentioned "configure the default boot-mode" but the new version doesn't mention that?
> Perhaps it's just me, but "reset your EEPROMs" sounds like an odd phrase, and specifically mentioning the "USB controller EEPROM" feels like an unnecessary level of detail?
> I'm struggling to come up with anything "better" than the phrasing used by the original though 😕
> 
> "We recommend using Raspberry Pi Imager to update the bootloader (which restores the default configuration) and change the default boot mode."

Thanks - I hadn't noticed Imager seems to have grown a new feature. I will go with your suggested wording.

Reply:
I've had a look at Imager, and it seems you now need to postively select a particular boot order, so I've gone with a slightly different wording. I've also add a sentence to direct the user to where in Imager such an option resides, although I'm not sure this is actually necessary.

Reply:
As you've probably seen, it's not RPi Imager that has grown a new feature, just the JSON file that RPi Imager downloads that has gained some new options :wink: